client/msp/users/models.go

```diff
@@ -0,0 +1,26 @@
+package users
+
+type MspCreateUsersInput struct {
+	TenantUid string      `json:"tenantUid"`
+	Users     []UserInput `json:"users"`
+}
+
+type MspDeleteUsersInput struct {
+	TenantUid string   `json:"tenantUid"`
+	Usernames []string `json:"usernames"`
+}
+
+type UserInput struct {
```

Comment:
Why `UserInput` and not `UserDetails`? Threw me off for a second in the `MspCreateUsersInput`

Reply:
Good call. Will rename.

Reply:
Fixed